README.md

```diff
@@ -10,34 +10,43 @@ Please note that this plugin is under active development, and might not be ready
 
 ## Contributing
 
-If you're interested in contributing to the development of Prettier for Ruby, you can follow the [CONTRIBUTING guide from Prettier](https://github.com/prettier/prettier/blob/master/CONTRIBUTING.md), as it all applies to this repository too.
+If you're interested in contributing to the development of Prettier for Ruby:
 
-To test it out on a Ruby file:
+* Clone this repository
+* Run `yarn` to install dependencies
+* Ensure all tests run with `yarn run test`
 
-* Clone this repository.
-* Run `yarn`.
-* Create a file called `test.rb`.
-* Run `yarn prettier test.rb` to check the output.
-  <!--
+You will find it useful to read the [CONTRIBUTING guide from Prettier](https://github.com/prettier/prettier/blob/master/CONTRIBUTING.md),
+as it all applies to this repository too.
 
-## Install
+### Development
 
-````bash
-yarn add --dev --exact prettier @prettier/plugin-ruby
-``` -->
+Prettier-Ruby makes use of Ruby's official lexer and parser, [Ripper](https://docs.ruby-lang.org/en/2.5.0/Ripper.html).
+You can see the output of Ripper's tokens, sexpressions, the intermediate representation that prettier-ruby uses,
+as well as the final prettier-ruby result by appending `DEBUG=true` when running tests:
 
-## Configure
+```bash
+DEBUG=true yarn run test --watch
+```
 
-.prettierrc:
+You can run the Ruby AST Exporter directly with a given string too:
 
-```json
-{
-  "plugins": ["@prettier/plugin-ruby"]
-}
-````
+```
+$ ruby vendor/ruby/astexport.rb "puts 'hello world'"
+```
 
-## Use
+To run an individual test folder:
 
 ```bash
-prettier --write "**/*.rb"
+DBEUG=true yarn run test tests/defs --watch
 ```
+
+## Usage
+
+Until this package is officially released to npm you must clone it yourself, and run it directly:
+
+* Clone this repository
+* Run `yarn` to install dependencies
```

Comment:
Note: I've deleted the `package-lock.json` and kept the `yarn.lock` file as all references in the read me are to yarn, rather than npm.